enricher/ecs/ecs.go

```diff
@@ -55,13 +59,15 @@ func NewEnricher() *Enricher {
 // EnrichRecord modifies existing record.
 func (enr *Enricher) EnrichRecord(r map[interface{}]interface{}, t time.Time) map[interface{}]interface{} {
 	resource := map[interface{}]interface{}{
-		mappings.RESOURCE_ACCOUNT_ID: enr.canvaAWSAccount,
-		"service.name":               enr.canvaAppName,
-		"cloud.platform":             "aws_ecs",
-		"aws.ecs.launchtype":         "EC2",
-		"aws.ecs.task.family":        enr.ecsTaskFamily,
-		"aws.ecs.task.revision":      enr.ecsTaskRevision,
-		"aws.log.group.names":        enr.logGroup,
+		mappings.RESOURCE_ACCOUNT_ID:     enr.canvaAWSAccount,
+		mappings.RESOURCE_APPLICATION_ID: enr.canvaAppId,
```

Comment:
nit: it will be more consistent to use the same name everywhere
```suggestion
		mappings.RESOURCE_APPLICATION_ID: enr.canvaApplicationId,
```